.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
This odd formatting in GitHub is due to the official JSON specification not supporting comments (JSON wasn't intended to be used for configuration files), however VS Code's configuration file format allows them (it was VS Code that auto-filled this section in) - Happy to remove these lines if the help-text isn't useful

Reply:
Great! I like keeping that in there for people who aren't familiar with the feature (a.k.a. myself)